### PR TITLE
Add HP 41C

### PR DESCRIPTION
**Pull request by kcoder63**
hello world in HP 41C

### CODE REVIEW

**Review by Richienb — CHANGES_REQUESTED** [review]
Remember to update the readme and languages count

File: HP_41C [hunk 1]
```diff
@@ -0,0 +1,3 @@
+01 LBL "HELLO"
+02 "HELLO WORLD"
```
Comment (Richienb) [inline HP_41C:2]: Is it possible to change `HELLO WORLD` into `Hello World`?

Comment (kcoder63) [reply HP_41C:2]: I think no 

SAVE_20210123_124833

**Comment by Richienb** [comment]
Thanks!

**Pull request merged**